static/css/main.css

```diff
@@ -122,13 +122,36 @@ textarea {
   resize: vertical;
 }
 
+
 blockquote {
-  margin-left: 1rem;
   font-size: 1.2rem;
-  border-left: 2px solid #3700FF;
```

Comment:
Looks like this border is still defined for mobile devices, see line 545 of this file.

Reply:
Gah. I suck at CSS stuff :/

About moving `<cite>` higher, I agree and we could then have it perhaps aligned to the right, like a quotation?

Reply:
I basically copied/pasted the same style for mobile.

Reply:
Hm... something happened that now on mobile devices, there is horizontal scrolling.
Can try to fix that, would you mind if I commit to this branch?

Reply:
Took the liberty and pushed. Please pull new commit before pushing again.

Also, changed placement of `<cite>`. Let me know if you think it's OK.

Reply:
Wait. Since you committed to the same branch, why do I need to pull?